maps/canada-britishColumbia-yaletown.md

```diff
@@ -7,5 +7,5 @@ years_published: 1967
 location: Location
 nosheets: 6
 infourl: http://resolve.library.ubc.ca/cgi-bin/catsearch?bid=2802600
-geojsonurl: https://github.com/ubc-lib-geo/spatial-indexes/blob/master/canada_britishColumbia_yaletown.geojson
+geojsonurl: https://ubc-lib-geo.github.io/spatial-indexes/north-america/canada_britishColumbia_yaletown.geojson
```

Comment:
The file URL was changed in the last commit, so it's fixed on develop already. This change can be safely removed.